P5/Exemplars/tei_rdfa.odd

```diff
@@ -0,0 +1,227 @@
+                of conditions and the following disclaimer.</item>
+              <item>Redistributions in binary form must reproduce the above copyright notice, this
+                list of conditions and the following disclaimer in the documentation and/or other
+                materials provided with the distribution.</item>
+            </list>
+            <p>This software is provided by the copyright holders and contributors "as is" and any
+              express or implied warranties, including, but not limited to, the implied warranties
+              of merchantability and fitness for a particular purpose are disclaimed. In no event
+              shall the copyright holder or contributors be liable for any direct, indirect,
+              incidental, special, exemplary, or consequential damages (including, but not limited
+              to, procurement of substitute goods or services; loss of use, data, or profits; or
+              business interruption) however caused and on any theory of liability, whether in
+              contract, strict liability, or tort (including negligence or otherwise) arising in any
+              way out of the use of this software, even if advised of the possibility of such
+              damage.</p>
+          </licence>
+          <p>TEI material can be licensed differently depending on the use you intend to make of it.
+            Hence it is made available under both the CC+BY and BSD-2 licences. The CC+BY licence is
+            generally appropriate for usages which treat TEI content as data or documentation. The
+            BSD-2 licence is generally appropriate for usage of TEI content in a software
+            environment. For further information or clarification, please contact the <ref
+              target="mailto:info@tei-c.org">TEI Consortium</ref>. </p>
+        </availability>
+      </publicationStmt>
+      <sourceDesc>
+        <p>born digital</p>
+      </sourceDesc>
+    </fileDesc>
+  </teiHeader>
+  <text>
+    <body>
+      <schemaSpec ident="tei_rdfa" start="TEI">
+        <moduleRef key="tei"/> <!-- required -->
+        <moduleRef key="header"/> <!-- required -->
+        <moduleRef key="core"/> <!-- required -->
+        <moduleRef key="textstructure"/> <!-- required -->
+        <moduleRef key="gaiji"/>
+        <moduleRef key="verse"/>
+        <moduleRef key="drama"/>
+        <moduleRef key="spoken"/>
+        <moduleRef key="cmc"/>
+        <moduleRef key="dictionaries"/>
+        <moduleRef key="msdescription"/>
+        <moduleRef key="transcr"/>
+        <moduleRef key="textcrit"/>
+        <moduleRef key="namesdates"/>
+        <moduleRef key="figures"/>
+        <moduleRef key="corpus"/>
+        <moduleRef key="linking"/>
+        <moduleRef key="analysis"/>
+        <moduleRef key="iso-fs"/>
+        <moduleRef key="nets"/>
+        <moduleRef key="certainty"/>        
+        <moduleRef key="tagdocs" except="elementSpec moduleRef schemaSpec" />
```

Comment:
My instinct is that this should be
~~~xml
<moduleRef key="tagdocs" include="att code eg egXML gi ident tag val"/>
~~~